lib/kafo/kafo_configure.rb

```diff
@@ -312,33 +312,33 @@ def app_option(*args, &block)
 
     def set_app_options
       app_option ['--[no-]colors'], :flag, 'Use color output on STDOUT',
-                            :default => !!config.app[:colors]
+                            :default => !!config.app[:colors], :advanced => true
       app_option ['--color-of-background'], 'COLOR', 'Your terminal background is :bright or :dark',
-                            :default => config.app[:color_of_background]
+                            :default => config.app[:color_of_background], :advanced => true
       app_option ['--dont-save-answers'], :flag, "Skip saving answers to '#{self.class.config.answer_file}'?",
-                            :default => !!config.app[:dont_save_answers]
+                            :default => !!config.app[:dont_save_answers], :advanced => true
       app_option '--ignore-undocumented', :flag, 'Ignore inconsistent parameter documentation',
-                            :default => false
+                            :default => false, :advanced => true
       app_option ['-i', '--interactive'], :flag, 'Run in interactive mode'
       app_option '--log-level', 'LEVEL', 'Log level for log file output',
-                            :default => config.app[:log_level]
+                            :default => config.app[:log_level], :advanced => true
       app_option ['-n', '--noop'], :flag, 'Run puppet in noop mode?',
                             :default => false
       app_option ['-p', '--profile'], :flag, 'Run puppet in profile mode?',
-                            :default => false
+                            :default => false, :advanced => true
```

Comment:
I noticed some of these are split across multiple lines while others aren't. Could we make the style more consistent either way while we are here?

Reply:
How's this update look?